src/main/java/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,52 @@
+package dao;
+
+import db.Storage;
+import exceptions.WrongPasswordException;
+import java.util.List;
+import java.util.NoSuchElementException;
+import lib.Dao;
+import model.Bet;
+import model.User;
+
+@Dao
+public class UserDaoImpl implements UserDao {
+    private List<User> storage = Storage.userStorage;
+    private User user;
```

Comment:
this should be a local variable, if it will be needed

Reply:
done